my-uw/index.md

```diff
@@ -78,6 +78,219 @@ slug: my-uw
   <h4>App Description</h4>
   <p>80 characters maximum. Avoid the word “your.” (For example, instead of “view your earnings and tax statements,” use “view earnings and tax statements.”)</p>
   
+
+  <div class="page-header">
+    <h2>Widgets</h2>
+  </div>
+  <p>MyUW displays a grid of widgets on the home page in widget mode. You can easily create widgets using the <a href="https://github.com/UW-Madison-DoIT/myuw-smart-widget-creator">Widget Creator</a>. Each widget follows approximately this structure, but <i>widget developers only need to worry about the code inside the widget-body div.</i></p>
+  {% highlight html %}
+  <div class="widget-frame">
+    <div class="widget-header">
+      <div class='widget-info'>
+        <i class="fa fa-info-circle" ... ></i>
+      </div>
+      <div class='widget-remove'>
+        <i class="fa fa-times" ... ></i>
+      </div>
+      <div class="widget-title">
+        <h4>My Courses Services</h4>
+      </div>
+    </div>
+    <div class="widget-body">
+      <!-- Your widget content here -->
+      <a class="btn btn-default launch-app-button">See All</a>
+    </div>
+  </div>
+  {% endhighlight %}
+  <h4>When to use</h4>
+  <p>Widget template content is a great way to get some content right on the homepage.  You can show static content that is the same for all users or show data that is supplied via a json feed from the portal. It is not useful to use widget content to only show a link that directly links to the portlet.  Reserve the widget template for more rich displays of content.  Also most web proxy portlets are not a good candidate for widget templating.  Constructing specific urls for specific users without a json feed is not what the template can do.</p>
+  <div class="row">
+    <div class="col-xs-4">
+      <div class="widget-frame" id="portlet-id-{{::portlet.nodeId}}">
+        <div class="widget-header">
+          <div class='widget-info'>
+            <i title="Info" class="fa fa-info-circle"
+            tooltip="{{::portlet.description}}"
+            tooltip-trigger="mouseenter"
+            tooltip-placement="top"
+            tooltip-popup-delay="200"></i>
+          </div>
+          <div class='widget-remove'>
+            <i title="Remove" class="fa fa-times portlet-options" ng-click="widgetCtrl.removePortlet(portlet.nodeId, portlet.title)"></i>
+          </div>
+          <div class="widget-title">
+            <h4>My Courses (6)</h4>
+          </div>
+          <ul class="widget-list">
+            <li>
+              <p class="bold">General Physics</p>
```

Comment:
`bold` sounds like a class that's about how content is presented, instead of about what the content is.  Would it be better to invent some kind of `list-item-title` or `title` or so class?